kafka/datadog_checks/kafka/data/metrics.yaml

```diff
@@ -73,71 +73,71 @@ jmx_metrics:
-              alias: kafka.producer.bufferpool_wait_time
-          - batch-size-avg:
-              metric_type: gauge
-              alias: kafka.producer.batch_size_avg
-          - batch-size-max:
-              metric_type: gauge
-              alias: kafka.producer.batch_size_max
-          - compression-rate-avg:
-              metric_type: gauge
-              alias: kafka.producer.compression_rate_avg
-          - record-queue-time-avg:
-              metric_type: gauge
-              alias: kafka.producer.record_queue_time_avg
-          - record-queue-time-max:
-              metric_type: gauge
-              alias: kafka.producer.record_queue_time_max
-          - request-latency-avg:
-              metric_type: gauge
-              alias: kafka.producer.request_latency_avg
-          - request-latency-max:
-              metric_type: gauge
-              alias: kafka.producer.request_latency_max
-          - record-send-rate:
-              metric_type: gauge
-              alias: kafka.producer.records_send_rate
-          - records-per-request-avg:
-              metric_type: gauge
-              alias: kafka.producer.records_per_request
-          - record-retry-rate:
-              metric_type: gauge
-              alias: kafka.producer.record_retry_rate
-          - record-error-rate:
-              metric_type: gauge
-              alias: kafka.producer.record_error_rate
-          - record-size-max:
-              metric_type: gauge
-              alias: kafka.producer.record_size_max
-          - record-size-avg:
-              metric_type: gauge
-              alias: kafka.producer.record_size_avg
-          - requests-in-flight:
-              metric_type: gauge
-              alias: kafka.producer.requests_in_flight
-          - metadata-age:
-              metric_type: gauge
-              alias: kafka.producer.metadata_age
-          - produce-throttle-time-max:
-              metric_type: gauge
-              alias: kafka.producer.throttle_time_max
-          - produce-throttle-time-avg:
-              metric_type: gauge
-              alias: kafka.producer.throttle_time_avg
+        bean_regex: 'kafka\.producer:type=producer-metrics,client-id=([-.\w]+)'
+        attribute:
+          waiting-threads:
+            metric_type: gauge
+            alias: kafka.producer.waiting_threads
+          buffer-total-bytes:
+            metric_type: gauge
+            alias: kafka.producer.buffer_bytes_total
+          buffer-available-bytes:
+            metric_type: gauge
+            alias: kafka.producer.available_buffer_bytes
+          bufferpool-wait-time:
+            metric_type: gauge
+            alias: kafka.producer.bufferpool_wait_time
+          batch-size-avg:
+            metric_type: gauge
+            alias: kafka.producer.batch_size_avg
+          batch-size-max:
+            metric_type: gauge
+            alias: kafka.producer.batch_size_max
+          compression-rate-avg:
+            metric_type: rate
```

Comment:
Why the change?

Reply:
Ah, looks like `metrics.yaml` and `conf.yaml.example` defined this metric differently. I guess I should change both to gauge